### ISSUE TITLE
Ship 7 OCR languages data in docker image

### ISSUE THREAD

**Issue by ciur** [ciur/papermerge#586]
**Info:**
 - Papermerge 3.0.2

Even though docs says:
```
By default the Papermerge Docker image includes English, German, French, Italian, Spanish, Romanian and Portugues OCR languages.
```
In [docker image only german and english](https://github.com/papermerge/papermerge-core/blob/master/docker/prod/Dockerfile#L26) OCR data is installed.

This ticket is about shipping French, Italian, Spanish, Romanian and Portugues OCR languages data by default.

This ticket was opened to fix problem mentioned in [this discussion](https://github.com/ciur/papermerge/discussions/585)

**Comment by ciur** [comment]
[Pull Request](https://github.com/papermerge/papermerge-core/pull/317)

**Comment by ciur** [comment]
Fix will be shipped in 3.0.3
